src/parse_logcfg.h

```diff
@@ -40,6 +40,7 @@ int parse_configfile(FILE *fp);
 int parse_logcfg(char *inputbuffer);
 
 extern char *error_details;
+extern int tune_seconds;
 
```

Comment:
Better declare variable needed 'during' parse_logcfg in the *.c file. 
But it should no longer be needed at all if you move it to globalsvars.h as suggested above.